### DISCUSSION TITLE
Custom color names?

### DISCUSSION THREAD

**Discussion by SebastianStehle** [saadeghi/daisyui#2801] [General]
Hi,

I really like your approach with variables and so on. I would like to add custom color that is also added as a variable with all the color space conversions that you do.

Is this somehow possible at the moment? I could just use tailwind colors, but they are not available as css variables.

**Comment by saadeghi** [comment]
This tutorial would be helpful:  
https://daisyui.com/blog/how-to-add-new-colors-to-daisyui/

Let me know if you have any questions
